put suffixes on all time points (#146)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

a solution might be built.  This list is not intended to be complete,
but is just representative enough to highlight various timing considerations.

All times are relative to the local clocks, indicated by an "a" (Attester),

> "v" (Verifier), or "r" (Relying Party) suffix.

Reviewer: Can you fix line 1236 to also have the quotes and capitalize Verifier there?

Author: Done.

Handles are different from nonces as they can be used more than once and can be used by more than one entity at the same time.
Handles are different from timestamps as they do not have to convey information about a point in time, but their reception creates that information.
The reception of a handle is similar to the event that increments a relative tickcounter.
Receipt of a new handle invalidates a previously received handle.

In this example, Evidence generation based on received handles always uses the current (most recent) handle.
As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.

> The time that represents this event is named Handle received: time(HR_x). Due to distribution over the network, there is some jitter with respect to time(HR_x) for each involved entity.

Reviewer: ```suggestion
Due to distribution over the network, there is some jitter with respect to the time the Handle is received, time(HR), for each involved entity.
```

> To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR_v') + offset - time(HR_v)`. The offset is typically based on a network's round trip time.

Reviewer: ```suggestion
To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR'_v) + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
```
The HR should be prime'ed rather than the subscript

Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR_v) - time(EG_a) < validity-duration`.

> From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "(time(HD')-time(HD)) - (time(HR_a)-time(EG_a)) \< validity-duration".

Reviewer: There is no event HD, only HR, and the time(HD) needs a subscript
```suggestion
From the point of view of a Verifier, the generation of valid Evidence is only possible, if the age of the handle used in the Evidence generation is younger than the duration of the distribution interval -- "(time(HR'_v)-time(HR_v)) - (time(HR_a)-time(EG_a)) \< validity-duration".
```
